AutoCollection/WebSnippet.ts

```diff
@@ -0,0 +1,146 @@
+import * as path from "path";
+import http = require("http");
+import fs = require('fs');
+
+import Logging = require("../Library/Logging");
+import TelemetryClient = require("../Library/TelemetryClient");
+
+class WebSnippet {
+
+    public static INSTANCE: WebSnippet;
+
+    private static _snippet: string;
+    private static _aiUrl: string;
+    private _isEnabled: boolean;
+    private _isInitialized: boolean;
+
+    constructor(client: TelemetryClient) {
+        if (!!WebSnippet.INSTANCE) {
+            throw new Error("Web snippet injection should be configured from the applicationInsights object");
+        }
+
+        WebSnippet.INSTANCE = this;
+        // AI URL used to validate if snippet already included
+        WebSnippet._aiUrl = " https://js.monitor.azure.com/scripts/b/ai.2";
```

Comment:
we should probably check for a pattern to see if our snippet is already there.  There could be multiple versions used, isn't it?  @hectorhdzg can you chime in?

Reply:
removed version numbers